NST/src/main/java/com/ebay/nst/NSTServiceWrapperProcessor.java

```diff
@@ -23,12 +26,15 @@
 import com.ebay.service.protocol.http.NSTHttpResponse;
 import com.ebay.utility.service.ServiceUtil;
 
+import javax.validation.constraints.NotNull;
+
 public final class NSTServiceWrapperProcessor {
 
 	private boolean disableSchemaValidation = false;
 	private boolean disableRequestResponseLogging = false;
 	private boolean confirmSuccess = true;
 	private NSTHttpClient<NSTHttpRequest, NSTHttpResponse> client;
+	private Charset responseParsingCharset = StandardCharsets.UTF_8;
```

Comment:
Please add unit test to ensure default value is set to `StandardCharsets.UTF_8`

Reply:
Good call. Done.